### PR TITLE
Issues and procedures conflict solving

### PR DESCRIPTION
**Pull request by leio10**
* Custom views for each procedure and issue type.
* Allow to fix issues and reevaluate related procedures.
* Allow ban users to fix issues.

Pending to remove with_issues state for procedures and replace with manual issue creation.

### CODE REVIEW

File: app/admin/issue.rb [hunk 1]
```diff
@@ -5,62 +5,82 @@
...
+    end
+
+    def update
+      issue = resource
+      issue.assign_attributes permitted_params[:issue]
+
+      Issues::FixIssue.call(issue: issue, admin: current_admin) do
+        on(:invalid) { render :edit }
+        on(:error) do
+          flash[:error] = t("census.issues.action_message.error")
```
Comment (deivid-rodriguez) [inline app/admin/issue.rb]: I think this needs to be `flash.now[:error]` or it won't be rendered. We'll find out when this code path is tested :)

File: app/models/issue.rb [hunk 2]
```diff
@@ -25,8 +26,32 @@ def absent?
...
+
+    self.close_result ||= :fixed
+    self.closed_at = Time.zone.now
+    save!
+  end
+
+  def gone!
+    return if closed?
+
+    self.close_result = :gone
```
Comment (deivid-rodriguez) [inline app/models/issue.rb]: I see that `gone!` overwrites the `close_result` while `fix!` does not. Is it expected? I also see that there's no `spec/commands/issues/close_issue_spec.rb`, maybe that unit test could catch this (if there was a bug here at all).

**Comment by deivid-rodriguez** [comment]
Looks good! Should I regenerate the `decidim-census_connector` vcr cassettes in case that finds any problems?

File: app/models/issue.rb (hunk 2 above)
Comment (leio10) [reply app/models/issue.rb]: You are right, I've changed the `close_result` assignment in the `CloseIssue` to keep it consistent.

File: app/admin/issue.rb (hunk 1 above)
Comment (leio10) [reply app/admin/issue.rb]: Thanks! I've checked this in all the project code and improved tests to check it in my last commits.

**Pull request merged**